tailwind.config.js

```diff
@@ -2,7 +2,30 @@
 module.exports = {
   content: ["./src/**/*.{js,jsx,ts,tsx}"],
   theme: {
-    extend: {},
+    screens: {
+      sm: "360px",
+      // => @media (min-width: 360px) { large-mobile }
+
+      md: "428px",
+      // => @media (min-width: 768px) { small-tablet }
+
+      lg: "744px",
+      // => @media (min-width: 1024px) { large-tablet }
+
+      xl: "1024px",
+      // => @media (min-width: 1280px) { small-laptop }
+
+      xxl: "1512px",
+      // => @media (min-width: 1536px) { large-laptop }
+    },
+    container: {
```

Comment:
So this centers stuff in the `container` class? Do we need to add `container` anywhere in the code?